src/man/krb5.conf.man

```diff
@@ -487,6 +487,11 @@ attempt fails.
 If this flag is true, then an attempt to verify initial
 credentials will fail if the client machine does not have a
 keytab.  The default value is false.
+.TP
+\fBclient_aware_channel_bindings\fP
```

Comment:
Man pages are generated from rst files in doc (this one is doc/admin/conf_files/krb5_conf.rst).  It's sufficient to modify the rst file.